src/openads/infrastructure/appnexus/AppNexusConnector.js

```diff
@@ -1,16 +1,14 @@
 import Connector from '../../domain/connector/Connector'
 
-require('@schibstedspain/ast')
-
 export default class AppNexusConnector extends Connector {
-  constructor ({source, connectorData}) {
+  constructor ({source, connectorData, appNexusClient}) {
     super({
       source: source,
       adapter: connectorData.adapter,
       configuration: connectorData.configuration
     })
     this._member = this.configuration.member
-    this._appNexusClient = apntag
+    this._apnTag = appNexusClient
```

Comment:
All OK but I (my opinion) think that _appNexusClient was a better name that _apnTag for the internal variable, as telling in the Connector that the variable is apntag we're also telling which is the implementation that comes in the constructor.